rancher2/schema_global_role.go

```diff
@@ -39,6 +39,15 @@ func globalRoleFields() map[string]*schema.Schema {
 				Schema: policyRuleFields(),
 			},
 		},
+		"inherited_cluster_roles": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Computed:    true,
```

Comment:
Based on the [definition of computed](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed) I don't think this should be true, since users can set this as part of the input and the definition states:
> Computed is often used to represent values that are not user configurable or can not be known at time of terraform plan or apply, such as date of creation or a service specific UUID.

Reply:
That is a great point! Shouldn't the same logic apply to majority of other fields? 
```
* `description` - (Optional/Computed) Role template description (string)
* `role_template_ids` - (Optional/Computed) Inherit role template IDs (list)
* `rules` - (Optional/Computed) Role template policy rules (list)
* `annotations` - (Optional/Computed) Annotations for role template object (map)
* `labels` - (Optional/Computed) Labels for role template object (map)
```

Reply:
I'd agree that this should apply to most of the other fields. However, we can't copy a bad practice just because another area uses it. I would recommend looking into the history of those other fields - if you can find a justification for using the value as true I'll consider it, but absent that I don't think it makes sense to continue the pattern here.

Reply:
I agree there is no point in spreading it further. FYI I couldn't find any justifications in commits so far.